charts/flyte/values.yaml

```diff
@@ -237,6 +235,14 @@ flyte:
     # -- affinity for Flyteconsole deployment
     affinity: {}
 
+  #
+  # Common secret auth
+  #
+
+  adminOauthClientCredentials:
+    clientId: flytepropeller
```

Comment:
[This value](https://github.com/flyteorg/flyte/blob/484c5b79ba614a5401cc24ca302358c871002d44/charts/flyte/values.yaml#L452) should be driven from `.Values.adminOauthClientCredentials.clientId`

Scheduler should have a similar one, not sure where is it... or maybe it's referencing the same one?

Reply:
It is referencing the same one